### PR TITLE
chore: fix error assertions

### PR DESCRIPTION
**Pull request by brc-dd**
### `Notable Changes`

`.catch` + `match`/`equals` doesn't assert that a particular call throws error. It just asserts IF an error is thrown it will satisfy assertions inside. Consequently, it will pass the test even if something doesn't throw while it should.

### `Type`

- [x] Test
- [x] Chore

### `SemVer`

- [ ] No bump needed

### `Checklist`

- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes are merged and published in downstream modules

### CODE REVIEW

File: test/rc.test.js [hunk 1]
```diff
@@ -54,12 +54,3 @@ test('.postcssrc - {Object} - Process SSS', () => {
       })
   })
 })
-
-describe('Load Config Error', () => {
```
Comment (brc-dd) [inline test/rc.test.js:58]: moved to Errors.test.ts

File: test/Errors.test.js [hunk 2]
```diff
@@ -1,68 +1,77 @@
...
 
-test('Loading Config - {Error}', () => {
-  return postcssrc({}, 'test/err').catch(err => {
-    match(err.message, /^No PostCSS Config found in: (.*)$/)
+// FIXME: this is resolving to nearest postcss config (outside root)
+// test('Loading Config - {Error}', async () => {
+//   await rejects(() => postcssrc({}, 'test/err'), {
+//     message: /^No PostCSS Config found in: (.*)$/
+//   })
+// })
```
[the remark this answers is not in the document]
Comment (ai) [reply test/Errors.test.js:11]: Seems like this test never moved to `match` area (this is why `rejects` is better).

As I understand it should ignore error in broken config and return `No PostCSS Config found in`. What is current behaviour in this test?

[the remark this answers is not in the document]
Comment (brc-dd) [reply test/Errors.test.js]: Ah, currently there is no postcssrc file directly inside `test/err`. Running `postcssrc({}, 'test/err')` is not failing and returning content from `test/postcss.config.mjs`. Should it fail if there is no config directly inside the specified path?

[the remark this answers is not in the document]
Comment (ai) [reply test/Errors.test.js]: Let’s remove test. It is rare case and test didn’t work for a while.

[the remark this answers is not in the document]
Comment (brc-dd) [reply test/Errors.test.js]: Done.

**Comment by ai** [comment]
Thanks!

**Pull request merged**